contrib/scalapblib/src/ScalaPBModule.scala

```diff
@@ -33,6 +33,7 @@ trait ScalaPBModule extends ScalaModule {
 
   def scalaPBSingleLineToProtoString: T[Boolean] = T { false }
 
+  // ScalaPB enables lenses by default, this option allows you to disable it
```

Comment:
```suggestion
  /** ScalaPB enables lenses by default, this option allows you to disable it. */
```

Otherwise, it is only a code comment, but not ScalaDoc.